### DISCUSSION TITLE
extract the sulcal basins as a new annotation file

### DISCUSSION THREAD

**Discussion by HusBunia** [brainvisa/brainvisa.github.io#135] [Q&A]
Hi BrainVisa: 

Thank you for the new release of BrainVISA! I have a question regarding the extraction of sulcal lines and basins. My goal is to create a group-level sulcal basins atlas and represent the cortex using gyral/sulcal lines. I initially tried using AimSulcaLines to extract sulcal basins, but it only returned the coordinates of sulcal lines. I am wondering if there is a way to directly generate sulcal basins parcellation using BrainVISA? Moreover, assuming I have extracted sulcal basins for each individual, how can I construct a group-level basin atlas, given that the number of basins might differ among subjects? I would appreciate any suggestions or guidance you could provide.

Thank you!
Best regards, 
Hus

**Comment by oliviercoulon** [comment]
Hello Hus, 
there is no ready made process for extracting (and labelling , I assume) the sulcal basins. First, one way one can usually define the basins is: areas of negative mean curvature. You can threshold a curvature map to get that. If you want to label them, you can use the lmabels of slcal lines, it is a good idea. You can produce a texture of labelled sulcal lines using _cortical surface -> anatomy -> tools ->  Sulcal Lines Extraction_. But you will get only the sulca lines that are part of the Hip Hop model... To produce a group map you need a surface based registration (for instance the Hip Hop process in BrainVisa. Indeed, the variable nuùmber of basins is an issue. But at least, using the Sulcal Lines Extraction process, you will have only basins that exist across all subjects. 
I hope this helps !

Olivier

**Comment by gauzias** [comment]
Hi,
to complement the answer from Olivier, you can get a parcellation of the cortical suface in sulcal basins at the individual level using the pipeline for extracting the sulcal pits: _cortical surface -> anatomy -> Sulcal Pits Extraction_
Then regarding the construction of group-level basin labeling or atlas, this is a complex question.
You can read the following paper to better understand the problem:
https://www.sciencedirect.com/science/article/abs/pii/S1361841520301134?via%3Dihub

**Reply by HusBunia** [reply]
Hi gauzias :

Thanks for your suggestion. Sorry for the late reply.  I believe the method outlined in your suggested paper is exactly what I need for my project. Specifically, I am aiming to create a projection from cortical parcellation to the pial outer-surface, with the assumption that the sulcal basins would maximize the projection area on the outer-surface. I did notice that the paper provides two different atlases based on adult and child datasets. I was wondering if these atlases could be shared ?  Since It is not easy to reproduce your algorithm exactly.  
Thanks again for all of the reply !
Best regards,
Hus

**Reply by gauzias** [reply]
Hi,
all the tools to build your own atlas for your specific dataset are available in brainvisa: 
cortical_surface/processes/anatomy/tools/PitsTextureSmoothing
cortical_surface/processes/anatomy/group/ProjectTextureOntoAtlas
cortical_surface/processes/anatomy/group/PitsDensityMap
cortical_surface/processes/anatomy/group/Group_watershed
I recommend to use these to compute your own atlas, which will match your population better than external an atlas.
Best

**Reply by HusBunia** [reply]
Greatly thanks for your help !!  I have spent some time on this algorithm and it will save me a lot of time.
